File: chroma-league/src/test/java/com/bonepl/chromaleague/hud/animations/AllyMountainDragonKillAnimationTest.java
package com.bonepl.chromaleague.hud.animations;

import com.bonepl.chromaleague.hud.AnimationTester;
import org.junit.jupiter.api.Test;

class AllyMountainDragonKillAnimationTest {

    @SuppressWarnings("JUnitTestMethodWithNoAssertions")
    @Test
    void playAllyMountainDragonKillAnimation() {
        new AnimationTester().testAnimation(new AllyMountainDragonKillAnimation());
    }
}

File: test/hud/animations/AllyMountainDragonKillAnimation.test.ts
import {describe, test} from 'vitest';
import {AnimationTester} from '../../AnimationTester.js';
import {AllyMountainDragonKillAnimation} from '../../../src/hud/animations/AllyMountainDragonKillAnimation.js';

/**
 * Integration test for ally mountain dragon kill animation on actual hardware.
 */
describe('AllyMountainDragonKillAnimation Integration', () => {
    test('playAllyMountainDragonKillAnimation', async () => {
        await new AnimationTester().testAnimation(new AllyMountainDragonKillAnimation());
    }, 30000);
});
